README.md

```diff
@@ -152,7 +152,8 @@ Interface:
 class CLI {
     constructor(opts: {
         input?: NodeJS.ReadableStream,
-        output?: NodeJS.WritableStream
+        output?: NodeJS.WritableStream,
+        quietBlank?: boolean
```

Comment:
```suggestion
        /** Set to true to prevent cliffy from displaying help text after entering a blank line. Defaults to false */
        quietBlank?: boolean
```